experimental/direct_win/README.md

```
@@ -0,0 +1,90 @@
+npm -v
+```
+
+* 未インストールの場合は、[Node.js LTS](https://nodejs.org/ja) をインストールしてください。
+
+#### 4. Python の確認（server用）
+
+```
+python --version
+```
+
+* バージョンは 3.12 以上を推奨。
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: dash; Actual: asterisk

(MD004, ul-style)